### ISSUE TITLE
Version 1.4 has a heavy network load in Rancher

### ISSUE THREAD

**Issue by rayout** [containous/traefik#2255]
I will show in the screenshots.

Version 1.3.8:
![1 3 8](https://user-images.githubusercontent.com/2248162/31485831-e605f8f4-af3d-11e7-84c9-b8dd541a593d.png)

Version v1.4.0-rc5:
![img-2017-10-12-11-04-03](https://user-images.githubusercontent.com/2248162/31485820-db474210-af3d-11e7-8718-9f5d6251dbb0.png)

My traefik.toml:

```toml
debug = false
logLevel = "INFO"
traefikLogsFile = "/opt/traefik/log/traefik.log"
accessLogsFile = "/opt/traefik/log/access.log"
InsecureSkipVerify = false
defaultEntryPoints = ["http", "https"]
[entryPoints]
  [entryPoints.http]
  address = ":80"
  compress = true
  [entryPoints.https]
  address = ":443"
  compress = true
    [entryPoints.https.tls]
      [[entryPoints.https.tls.certificates]]
      certFile = "/opt/traefik/certs/traefik.crt"
      keyFile = "/opt/traefik/certs/traefik.key"

[web]
address = ":8000"
ReadOnly = false
[web.statistics]
  RecentErrors = 10

[rancher]
domain = "dev.rz"
Watch = true
RefreshSeconds = 15
ExposedByDefault = false
EnableServiceHealthFilter = false

[rancher.metadata]
IntervalPoll = true
Prefix = "/2016-07-29"

[file]
filename = "/opt/traefik/etc/rules.toml"
watch = true

[acme]
email = "develop@kodeks.ru"
storage = "/opt/traefik/acme/acme.json"
onDemand = true
OnHostRule = true
entryPoint = "https"
caServer = "https://acme-staging.api.letsencrypt.org/directory"

```

**Comment by SantoDE** [comment]
Hi @rayout,

depending on your set of stacks & services, that might be okay. We had some issues with colliding services based on the name, so we're now fetching more information from the API to prevent that from happening. It would be interesting to see the network load if you switch to API mode. If that's more or less equal (what I suspect will be the case), all should be fine :)

**Comment by rayout** [comment]
When i use rancher api on 1.4.0-rc5:

![1 4-api](https://user-images.githubusercontent.com/2248162/31487155-783dbea6-af42-11e7-8f57-b5d9b0b6d827.png)

**Comment by SantoDE** [comment]
And there we go that my suspect was false :) Can you try with rc3 and rc4? So we can figure out where the regretion might have happened.

**Comment by rayout** [comment]
I tested all versions (rc1-rc5) with docker image [alpine-traefik](https://github.com/rawmind0/alpine-traefik) Everywhere the same thing. Network > 6Mbps

**Comment by SantoDE** [comment]
This is not "our" image. The integration made there is a custom Integration made by rawmind. Can you please check with our versions?

https://hub.docker.com/r/library/traefik/tags/
traefik:v1.4.0-rc5
traefik:v1.4.0-rc4
traefik:v1.4.0-rc3
traefik:v1.4.0-rc2
traefik:v1.4.0-rc1

**Comment by rayout** [comment]
I know that is not yours image. But inside it use https://hub.docker.com/r/library/traefik
alpine-traefik add only config generator.

But I'll try with your images, it just takes more time

**Comment by SantoDE** [comment]
Maybe just try with the latest rc :) So we can go sure it happens again.

**Comment by SantoDE** [comment]
Ah, before we move on I have an idea. As you're running the custom integration you are having 2 different providers (file and rancher-metadata) and both are basically doing the same and fetching the same.

To be sure that this is not the reason for that behaviour, can you please only enable the metadata provider when you test it with our images? @rayout

**Comment by rayout** [comment]
I remove [file] section and nothing changes =(
Will try with official image

**Comment by SantoDE** [comment]
Hey @rayout ,

I can confirm the issue. I've provided a potential fix in #2259.

Thanks for reporting! :)

**Comment by rayout** [comment]
Now i use official images 1.4.0-rc1 and 1.4.0-rc5
But i just run with command (It is difficult to slip the configuration file in the Rancher)

   ` command: --web --web.address=:8080 --rancher --rancher.metadata.prefix="/2016-07-29" --rancher.metadata.intervalpoll=true --rancher.watch=true --rancher.domain=rancher.localhost --rancher.endpoint=http://rancher.dmz/v1/projects/1a139 --rancher.accesskey=179394EFDFB70841D088 --rancher.secretkey=mW1VxKrFzUwKAeGaNbVarZ7pamsoyS54EAHKGHuW
`

When i add `rancher.metadata.intervalpoll=true` network up to 6-7 mbps

Thank you for fast reaction!

**Comment by SantoDE** [comment]
You're welcome. Please have a look at my previous post and give mx PR a test :)

**Comment by ldez** [comment]
Fixed by #2259
